### PR TITLE
chore(e2e): add fail fast flag

### PR DESCRIPTION
**Pull request by Lou1415926**
...turns out it's probably just as simple as that

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: e2e/e2e.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@ done
 echo $DOCKERHUB_TOKEN | docker login --password-stdin -u $DOCKERHUB_USERNAME
 
 #Run all the e2e tests
-cd /github.com/aws/copilot-cli/e2e/$TEST_SUITE && /go/bin/ginkgo -v -r --timeout=1h30m
\ No newline at end of file
+cd /github.com/aws/copilot-cli/e2e/$TEST_SUITE && /go/bin/ginkgo -v -r --timeout=1h30m --fail-fast
```
Comment (efekarakus) [inline e2e/e2e.sh:26]: will cleanup still run once a test aborts?

Comment (Lou1415926) [reply e2e/e2e.sh:26]: yes, `AfterSuite` wil run

**Review by dannyrandall — APPROVED** [review]
lol 🚀

**Review by bvtujo — APPROVED** [review]
lol; lmao 😭 😎

**Pull request merged**